dockerfile-image-update/src/main/java/com/salesforce/dockerfileimageupdate/utils/Constants.java

```diff
@@ -36,5 +36,6 @@ private Constants() {
     public static final String GIT_REPO_EXCLUDES = "excludes";
     public static final String GIT_API_SEARCH_LIMIT = "ghapisearchlimit";
     public static final String SKIP_PR_CREATION = "skipprcreation";
+    public static final String IGNORE_PR_COMMENT = "x";
```

Comment:
```suggestion
    public static final String IGNORE_IMAGE_STRING = "x";
```

Reply:
The SKIP_PR_CREATION might get confused with this one. I think ignore image string makes a little more sense. Open to better naming of this though.

Reply:
Updated to IGNORE_IMAGE_STRING